docs/configuration/README.md

```diff
@@ -663,23 +663,11 @@ The `limits_config` block configures global and per-tenant limits for ingesting
 logs in Loki.
 
 ```yaml
-# Per-user ingestion rate limit in samples per second.
-[ingestion_rate: <float> | default = 25000]
+# Per-user ingestion rate limit in samples per second. Units in MB.
```

Comment:
Lets match it to CLI help text since the limit is enforced on sample size not count.
```suggestion
# Per-user ingestion rate limit in sample size per second. Units in MB.
```